bec_alerts/models.py

```diff
@@ -31,7 +31,7 @@ class Issue(models.Model):
     last_seen = models.DateTimeField(null=True, default=None)
     module = models.CharField(max_length=255, default='')
     stack_frames = JSONField(default=list)
-    message = models.CharField(max_length=255, default='')
+    message = models.TextField(max_length=255, default='')
```

Comment:
You're changing this to a `TextField` to get around `max_length` issues, but this still has a `max_length`. I don't think I understand what's going on. How does this help?

Reply:
Whoops.